### ISSUE TITLE
V3B2 : Bug data from different pomps

### ISSUE THREAD

**Issue by robertrub** [nightscout/AndroidAPS#815]
[AndroidAPS_LOG_1635614248566.log.zip](https://github.com/nightscout/AndroidAPS/files/7447449/AndroidAPS_LOG_1635614248566.log.zip)
I deactivated the ending EROS and activated a new EROS. Process between 9:12 till 19:17. 
I got this error message once, validated and am continuing using the EROS pod normally till now.
Before deactivating EROS, I had activated a DASH with the same phone in parallel. I'm not sure if they are related.

![20211030_191715](https://user-images.githubusercontent.com/76626474/139555439-3c65e778-720f-4933-8222-e4423fbd6b7b.jpg)

**Comment by samspycher** [comment]
Hi Rob!

Thanks for reporting this. It is a duplicate of #782 and #775.

May I ask: did you experience any freezing of the screen while changing the pods?

**Comment by robertrub** [comment]
Hi @samspycher 

I was waiting for the freeze but no, nothing abnormal happened. 

I thought it had frozen as nothing was moving on the screen but I waited patiently and the next activation phase arrived without any intervention. I didn't get the Android "AAPS is not answering, want to close or wait ?" message. I think the process didn't took longer than usual but I'm not sure.

**Comment by robertrub** [comment]
Hi again @samspycher 

I changed my EROS pod just now (with the latest dev 258224055) and I had the freeze when I put the pod on my arm and hit the OK on the little popup message. All went OK with the EROS but I got the Android message "Wait or Close". I hit wait and AAPS came back to life and the pod is working OK. Then, I got the same "data coming from another pump" error message.

Attached is the logs. Freeze and message between 14:17 and 14: 20.

[AndroidAPS_LOG_1635859291848.log.zip](https://github.com/nightscout/AndroidAPS/files/7461010/AndroidAPS_LOG_1635859291848.log.zip)

**Comment by MilosKozak** [comment]
what did you click before the freeze?

**Comment by robertrub** [comment]
I hit the OK button, there

![image](https://user-images.githubusercontent.com/76626474/139919028-c6af0c1e-5aa6-424a-a966-0a7e979fe6c1.png)

and the "popup" disappeared, I was back to this page

![image](https://user-images.githubusercontent.com/76626474/139919153-df3ab4cc-08fc-41b8-ade5-4d9836e529ed.png)

**Comment by avereha** [comment]
I think we can close this and reopen if you still see the issue in dev/next beta.

The 'data from different pumps' should be fixed with https://github.com/nightscout/AndroidAPS/commit/85178ee19ff628dfe4422409973c6f8ae7b511b5

'screen freeze' should be fixed with https://github.com/nightscout/AndroidAPS/commit/52e343213249c0abff26aab4756b5c9edf8755c3
